### PR TITLE
Added note to install a package

### LINKED ISSUE #363: Docs - Build Section - Steps on Netlify missing note to install `zip-it-and-ship-it`
See: https://redwoodjs.com/docs/builds

```js
yarn rw build api
cd api
yarn zip-it-and-ship-it dist/functions/ zipballs/
```

Needs to include a note that the `zip-it-and-ship-it` package needs to be installed as a dev dependency.

And maybe include a link to the `AWS Serverless Deploy` for some more related info: https://redwoodjs.com/docs/deploy#aws-serverless-deploy.

The `yarn rw generate deploy aws_serverless` does install it as a api dev dependency:

```
  "devDependencies": {
    "@netlify/zip-it-and-ship-it": "^1.3.12",
```

### PR DESCRIPTION
**Pull request by lumenCodes**
Signed-off-by: cld <lumenluwiz@gmail.com>

This PR fixes #363 

I added the note to the build.md file

### CODE REVIEW

**Comment by lumenCodes** [comment]
@dthyresson Please check this out.

Thank you

**Review by dthyresson — CHANGES_REQUESTED** [review]
Thanks @lumenCodes for the additional note.

To clarify, should the 

> package needs to be installed as a dev dependency

in `api` or in the root Redwood app.

For example:

`yarn workspace api add @netlify/zip-it-and-ship-it`

or 

`yarn workspace add @netlify/zip-it-and-ship-it`

Also, a link to the package [https://www.npmjs.com/package/@netlify/zip-it-and-ship-it](https://www.npmjs.com/package/@netlify/zip-it-and-ship-it) would be helpful so that people can see what it is exactly what the package does.

This also show that that package is `@netlify/zip-it-and-ship-it` and not just `zip-it-and-ship-it`/
?

**Comment by lumenCodes** [comment]
> Thanks @lumenCodes for the additional note.
> 
> To clarify, should the
> 
> > package needs to be installed as a dev dependency
> 
> in `api` or in the root Redwood app.
> 
> For example:
> 
> `yarn workspace api add @netlify/zip-it-and-ship-it --dev`
> 
> or
> 
> `yarn add @netlify/zip-it-and-ship-it --dev`
> 
> Also, a link to the package https://www.npmjs.com/package/@netlify/zip-it-and-ship-it would be helpful so that people can see what it is exactly what the package does.
> 
> This also shows that that
> 
> 1. the package is `@netlify/zip-it-and-ship-it` and not just `zip-it-and-ship-it`
> 2. how to install as a [dev dependency](https://classic.yarnpkg.com/en/docs/cli/add/#toc-yarn-add-dev-d) in case people are not familiar with that option in yarn (--dev or -D)
>    ?

Ok thanks @dthyresson . I will look into these corrections and make changes. :ok_hand:

**Comment by lumenCodes** [comment]
@dthyresson I am not sure, but I think the package should be installed in the `api` . So I will `yarn workspace api add @netlify/zip-it-and-ship-it --dev`    

So I will just add

> Note: The `@netlify/zip-it-and-ship-it` package needs to be installed as a dev dependency in `api`.  
You can learn more about the package here: https://www.npmjs.com/package/@netlify/zip-it-and-ship-it .   For more information on `AWS Serverless Deploy` see the [docs here](https://redwoodjs.com/docs/deploy#aws-serverless-deploy).  
Also learn how to install in yarn (--dev or -D) as a dev [dependency](https://classic.yarnpkg.com/en/docs/cli/add/#toc-yarn-add-dev-d) here.

File: docs/builds.md [hunk 1]
```diff
@@ -14,7 +14,10 @@ cd api
 yarn zip-it-and-ship-it dist/functions/ zipballs/
 ```
 
-Each lambda function in `./api/dist/functions` is parsed by zip-it-and-ship-it resulting in a zip file per lambda function that contains all the dependencies required for that lambda function.
+Each lambda function in `./api/dist/functions` is parsed by zip-it-and-ship-it resulting in a zip file per lambda function that contains all the dependencies required for that lambda function.   
+Note: The `@netlify/zip-it-and-ship-it` package needs to be installed as a dev dependency in `api`.
+You can learn more about the package [here:] (https://www.npmjs.com/package/@netlify/zip-it-and-ship-it) . For more information on AWS Serverless Deploy see the [docs](https://redwoodjs.com/docs/deploy#aws-serverless-deploy) here.
+Also learn how to install in yarn (--dev or -D) as a dev dependency [here](https://classic.yarnpkg.com/en/docs/cli/add/#toc-yarn-add-dev-d).
```
Comment (thedavidprice) [inline docs/builds.md:20]:
```suggestion
Each lambda function in `./api/dist/functions` is parsed by zip-it-and-ship-it resulting in a zip file per lambda function that contains all the dependencies required for that lambda function.
   
>Note: The `@netlify/zip-it-and-ship-it` package needs to be installed as a dev dependency in `api/`. Use the command `yarn workspace api add -D @netlify/zip-it-and-ship-it`.
>- You can learn more about the package [here](https://www.npmjs.com/package/@netlify/zip-it-and-ship-it). 
>- For more information on AWS Serverless Deploy see these [docs](https://redwoodjs.com/docs/deploy#aws-serverless-deploy).
```

I've added the installation command in the Note, used the "blockquote" formatting for the Notes section, added bullets and fixed link issue, and removed the information about installing as a dependency.

@lumenCodes you can accept this commit directly or make changes yourself. Also, once you've made a commit, you can preview the changes on the Website using the netlify/redwoodjs/deploy-preview link in the GitHub Checks.

Screen Shot 2020-10-10 at 9 48 33 AM

Comment (lumenCodes) [reply docs/builds.md:20]: Thank you so much @thedavidprice I am really grateful. 
I think it is all fine to me :star_struck:

**Comment by thedavidprice** [comment]
Thanks @lumenCodes!

**Pull request merged**